modules/rsyslog/templates/tail-hilary.sh.erb

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Tail the logs outputted by all the Hilary application servers
+# Options:
+#   -n K    output the last K lines, instead of the last 10; or use -n +K to output lines starting with the Kth
+#   -l L    output lines of at least this level. Defaults to 30. For reference: 30=info, 40=warn, 50=error
+#
+
+# Reset in case getopts has been used previously in the shell
+OPTIND=1
+
+# Get the arguments
+lines=10
+level=30
+while getopts "n:l:" opt; do
+    case "$opt" in
+    n)  lines=$OPTARG
+        ;;
+    l)  level=$OPTARG
+        ;;
+    esac
+done
+
+shift $((OPTIND-1))
+
+# Hilary logs to local0-2015-03-26.log
+today_log="local0-$(date +%Y-%m-%d).log"
+
+# Generate a shitty regex that we can use to grep out log lines of a certain level
+regex='level":'$level
+level=$(($level + 10))
+while [ $level -lt 51 ] ; do
+    regex=$regex'\|level":'$level
+    level=$(($level + 10))
+done
```

Comment:
I wonder if we can just incorporate ./filter-bunyan into this in such a way that we could do something like:

`tail-hilary.sh -l .. -n .. -b "-l error"`

Where the `-b` flag is simply a bunyan filter per https://github.com/trentm/node-bunyan#cli-usage.

I can't recall all the stream "magic" that was required to get it into bunyan properly, though. It may be less robust than it's worth. But something like this might be worth it:

`tail-hilary.sh -b "-c 'this.name === \"oae-search\"'"`

Or it might be better to find a way to go in a direction that avoids such a quoting nightmare.